lang/cabal/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                cabal
 version             3.10.2.0
-revision            2
+revision            3
```

Comment:
no need to increase the `revision` - these are only affecting build dependencies.

Reply:
Thanks, fixed.